### PR TITLE
fix up all British spellings

### PR DESCRIPTION
**Pull request by xginn8**
tests: add test-spelling.sh

Closes: https://github.com/balena-io/docs/issues/1300
Change-type: patch
Signed-off-by: Matthew McGinn <matthew@balena.io>

### CODE REVIEW

**Comment by xginn8** [comment]
mostly a quick Hack Friday, but open to comments and suggestions :)

**Comment by garethtdavies** [comment]
This is bloody fantastic 😄

**Comment by xginn8** [comment]
> This is bloody fantastic smile

❯ echo "This is bloody fantastic"  | ./node_modules/american-british-english-translator/bin/english-translate 
1: 
  - 
    bloody: 
      issue:   Different meanings
      details: 
        American English: considered a euphemism for more emphatic swear words
        British English: 
          """
            expletive attributive used to express anger ("bloody car") or shock 
            ("bloody hell"), or for emphasis ("not bloody likely") (slang, today only 
            mildly vulgar) (similar US: damn ("damn car"))
          """
        Common:           having, covered with or accompanied by blood

**Comment by xginn8** [comment]
I should probably modify this to capture all output rather than short-circuiting, will update that.

File: shared/overview/1.x.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The [{{ $names.company.lower }}][balena] software running on your device leverag
 
 The operating system we install on your device is [Yocto][yocto] [Linux][linux]. This is an innovative distribution which comprises components which can be combined together to build a custom Linux system containing only the software you need for any target architecture. We plan to support additional Operating Systems bases in the future.
 
-This is perfect for {{ $names.company.lower }} - we have built an optimised Linux system for the devices we support and can guarantee the same behaviour on any further devices we support in the future.
+This is perfect for {{ $names.company.lower }} - we have built an optimizedd Linux system for the devices we support and can guarantee the same behavior on any further devices we support in the future.
```
Comment (lurch) [inline shared/overview/1.x.md]: I'm not American, but you probably don't want two 'd's in optimizedd ? :wink:

File: shared/overview/2.x.md [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 
-{{ $names.os.upper }} is an operating system built for easy portability to multiple device types (via the [Yocto framework](https://www.yoctoproject.org/)) and optimised for Linux Containers, and Docker in particular. There are many decisions, large and small, we have made to enable that vision, which are present throughout our architecture.
+{{ $names.os.upper }} is an operating system built for easy portability to multiple device types (via the [Yocto framework](https://www.yoctoproject.org/)) and optimizedd for Linux Containers, and Docker in particular. There are many decisions, large and small, we have made to enable that vision, which are present throughout our architecture.
```
Comment (lurch) [inline shared/overview/2.x.md]: ...maybe there's a bug in you dictionary??

File: shared/resinSync/_default.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ Once the CLI is installed globally, login with your {{ $names.company.lower }} a
 ```
 $ {{ $names.company.lower }} login
 ```
-You should then be presented with 3 options to login. The recommended method is `Web authorisation` which will open a dialog in your web browser and ask you to authorise the use of the CLI.
+You should then be presented with 3 options to login. The recommended method is `Web authorization` which will open a dialog in your web browser and ask you to authorize the use of the CLI.
```
Comment (lurch) [inline shared/resinSync/_default.md]: Minor nitpick, but IMHO you should check whether balena-cli displays `Web authorisation` or `Web authorization`

Comment (xginn8) [reply shared/resinSync/_default.md]: Yep, see https://github.com/balena-io/balena-cli/pull/1555 where I did the same for the CLI.

**Pull request merged**